doc/samples/keepalived.conf.IPv6

```diff
@@ -4,54 +4,49 @@ global_defs {
    router_id LVS_DEVEL
 }
 
-
 virtual_server_group IPv6_group {
-	ae00::2-9	80 
-	ae00::1		80 
+    ae00::2-9 80
+    ae00::1   80
 }
 
 #virtual_server group IPv6_group {
 virtual_server ae00::1 80 {
     delay_loop 6
-    lb_algo rr 
+    lb_algo rr
     lb_kind NAT
     persistence_timeout 50
     protocol TCP
 
-#    real_server ae00::10 80 {
+    # real_server ae00::10 80 {
```

Comment:
Please leave comment characters at the start of the line.

This applies also to several changes below too.